templates/campaign-url-builder/index.html

```diff
@@ -92,5 +92,10 @@ <h2 id="related-resources">Related resources</h2>
 
 {% block foot_scripts %}
 {{ super() }}
+<script>
+  ((function() {
+    window.BITLY_CLIENT_ID = {{ site.bitly_client_id | tojson | safe }};
```

Comment:
I really hate this but I don't see a better way to do it. I'm open to suggestions.